### PR TITLE
Add Dutch translation for Windows installer.

### PR DESCRIPTION
**Pull request by jerogee**

### CODE REVIEW

File: windows/translate/installerlng.nsi [hunk 1]
```diff
@@ -10,3 +10,6 @@
 
 ; !insertmacro MUI_LANGUAGE "Italian"
 ; !include "translate\it.nsi" ; include italian
+
+!insertmacro MUI_LANGUAGE "Dutch"
```
Comment (ann0see) [inline windows/translate/installerlng.nsi:14]: I'm not sure if "Dutch" is correct as language/value here? I will try to compile everything and see if it works ;-)

File: windows/translate/nl.nsi [hunk 2]
```diff
@@ -0,0 +1,37 @@
+; Dutch translation
+
+LangString DESKTOP_SET_SHORTCUT ${LANG_DUTCH} \
+    "Desktop shortcut maken"
+
+LangString INVALID_FOLDER_MSG ${LANG_DUTCH} \
+    "De doelmap bestaat al. Kies een nieuwe doelmap."
+
+LangString RUNNING_APP_MSG ${LANG_DUTCH} \
+    "${APP_NAME} loopt nog. Sluit het programma en start de installatie opnieuw.."
```
Comment (ann0see) [inline windows/translate/nl.nsi:10]: Are two dots correct here?

File: windows/translate/nl.nsi [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+LangString OLD_WRONG_VER_FOUND ${LANG_DUTCH} \
+    "We hebben een oudere versie van ${APP_NAME} in de 32 Bit Program Files map gevonden. Deze kan het beste worden verwijderd alvorens een nieuwe versie van ${APP_NAME} te installeren. Wilt u het nu laten verwijderen?"
+
+LangString OLD_WRONG_VER_FOUND_CONFIRM ${LANG_DUTCH} \
+    "Als u doorgaat zonder het te verwijderen, kan de installatie mogelijk mislukken! Weet u zeker dat u de oude versie niet wil verwijderen?"
+
+LangString OLD_VER_REMOVE_FAILED ${LANG_DUTCH} \
+     "FATALE FOUT: HET DE-INSTALLATIEPROGRAMMA HEEFT NIET GEWERKT. Zodra u op OK klikt zal de oude versie op uw PC blijven en proberen we de nieuwe versie te installeren. U kunt ook op annuleren klikken en het zelf proberen te verwijderen."
```
Comment (ann0see) [inline windows/translate/nl.nsi:19]: I think we should re-think this in english. At least my german background tells me that this sentence might need to be rephrased. @gilgongo

**Review by ann0see — CHANGES_REQUESTED** [review]
Great! Thanks. Just a few comments (especially the two dots might just be a typo)

Note to self: this hints me at the question: Should we move this folder to src/res/?

File: windows/translate/nl.nsi (hunk 3 above)
Comment (gilgongo) [reply windows/translate/nl.nsi:19]: I guess we could soften it a bit? So: 

"Sorry, we couldn't uninstall the old version. We'll try to install the new one, but you can also press cancel and try to remove the old version on your own."

File: windows/translate/installerlng.nsi (hunk 1 above)
Comment (ann0see) [reply windows/translate/installerlng.nsi:14]: Yes. It works. Great!

**Review by ann0see — APPROVED** [review]
Except for the two dots, this is ready.

File: windows/translate/nl.nsi (hunk 2 above)
Comment (jerogee) [reply windows/translate/nl.nsi:10]: Nice catch @ann0see. Should be single dot.

File: windows/translate/nl.nsi (hunk 3 above)
Comment (jerogee) [reply windows/translate/nl.nsi:19]: Yes, Dutch is generally quite "direct". I will soften it a bit whilst staying as close to the English as possible.

File: windows/translate/nl.nsi (hunk 2 above)
Comment (ann0see) [reply windows/translate/nl.nsi:10]: Great! Thanks

File: windows/translate/nl.nsi (hunk 3 above)
Comment (ann0see) [reply windows/translate/nl.nsi]: Agree. We should change this to gilgongo's proposal.

**Review by ann0see — APPROVED** [review]

File: windows/translate/nl.nsi (hunk 3 above)
Comment (ann0see) [reply windows/translate/nl.nsi]: @jerogee I think we should change it to gilgongo‘s proposal

**Review by ann0see — APPROVED** [review]
Thanks. Ready to be merged, I think?

File: windows/translate/nl.nsi (hunk 3 above)
Comment (jerogee) [reply windows/translate/nl.nsi]: Done

**Comment by ann0see** [comment]
@jerogee would it be ok if I include your work in a pr which includes this: https://github.com/jamulussoftware/jamulus/issues/944

**Comment by jerogee** [comment]
@ann0see: sure, go ahead!

**Comment by ann0see** [comment]
Great. I will close this PR then and add it to another one (including the changes of #949 )

**Pull request closed without merging**